src/test/java/org/intellij/sdk/codesync/database/SQLiteConnectionTest.kt

```diff
@@ -0,0 +1,45 @@
+package org.intellij.sdk.codesync.database
+
+import org.junit.jupiter.api.AfterEach
+import org.junit.jupiter.api.Test
+import java.io.File
+import java.sql.Connection
+import kotlin.test.assertFalse
+import kotlin.test.assertTrue
+
+class SQLiteConnectionTest {
+
+    @AfterEach
+    fun AfterEach(){
+        //1. Disconnect database
+        SQLiteConnection.getInstance().disconnect()
+
+        //2. Remove test db file
+        var file = File(CodeSyncTestUtils.getTestDBFilePath())
+        file.delete()
+    }
+
+    @Test
+    fun validateSQLiteConnection(){
```

Comment:
@syedgulahmed Can you point to me to this test?
https://github.com/codesyncapp/codesync-intellij/pull/97#discussion_r1233617665